src/PathDrawing/WayPoint.h

```diff
@@ -0,0 +1,37 @@
+// Copyright eeGeo Ltd (2012-2015), All Rights Reserved
+
+#pragma once
+
+#include "Types.h"
+#include "WayPointType.h"
+#include "space.h"
+#include <string>
+#include "LatLongAltitude.h"
+namespace ExampleApp
+{
+    namespace PathDrawing
+    {
+        class WayPoint : protected Eegeo::NonCopyable
+        {
+        private:
+            const int m_wpId;
+            const std::string& m_title;
```

Comment:
String is stored on this type as a const ref, but GetTitle() returns a copy - This seems the wrong way round?